chirp/drivers/retevis_rt98.py

```diff
@@ -586,7 +586,7 @@ def _ident(radio):
     ver_response = radio.pipe.read(16)
     LOG.debug(util.hexprint(ver_response))
 
-    verok, model, bandlimit = check_ver(radio, ver_response,
+    verok, model, bandlimit = check_ver(ver_response,
```

Comment:
Hmm, yeah @dforsi changed this recently in a big cleanup, but it seems clearly wrong. Maybe we can get a read on what the intent was? Definitely looks like that was wrong and this is right though.